ct_ansi.go

```diff
@@ -4,14 +4,21 @@ package ct
 
 import (
 	"fmt"
+	"os"
 	"strconv"
 )
 
 func resetColor() {
+	if os.Getenv("TERM") == "dumb" {
+		return
+	}
 	fmt.Print("\x1b[0m")
 }
 
 func ansiText(fg Color, fgBright bool, bg Color, bgBright bool) string {
+	if os.Getenv("TERM") == "dumb" {
```

Comment:
This one is ok. It does not break the assumption ansiText is a pure function of its parameters.